go.mod

```diff
@@ -75,7 +75,10 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
-require golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
+require (
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
+)
```

Comment:
Are you using these `sync` & `time` packages on purpose?

Reply:
`sync` - yes
`time` - Not explicitly. Is it possible that `sync` requires it?

Reply:
Moreover, if I'm not wrong, `time` was there before